app/controllers/users_controller.rb

```diff
@@ -0,0 +1,51 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
```

Comment:
We should have some kind of authorization scoping here. Pundit policies are pretty nice for this.

```suggestion
    @users = policy_scope(User.all)
```

Would call a policies/user_policy.rb scope and hide users the current user should not be able to see.